sfdc37/Package.md

```diff
@@ -1,102 +1,116 @@
-# Salesforce Connector
+Connects to Salesforce from Ballerina. 
 
-Allows connecting the Salesforce REST API.
+# Package Overview
 
+This package provides a Ballerina API for the Salesforce REST API. It provides the ability to do CRUD operations for 
+SObjects, query using SOQL, search using SOSL, describe SObjects and organizational data, etc. It handles OAuth 2.0 and 
+provides auto completion and type conversions.
 
-Salesforce connector provides a Ballerina API to access the Salesforce REST API. 
-It simplifies coding by handling OAuth 2.0 and performing type conversions.
-The following section provides you the details on how to use Ballerina Salesforce connector.
+**SObject Operations**
 
-## Compatibility
-
-| Ballerina Version         | API Version |
-| ------------------------- | ------------|
-|   0.970.0-beta15          |   v37.0     |
- 
-
-## Getting started
+The `wso2/sfdc37` package contains operations to do CRUD operations for standard and customized SObjects. It can create, 
+get, update, and delete SObjects via SObject IDs, and upsert via external IDs.
 
-1. Refer to [Getting Started guide](https://ballerina.io/learn/getting-started/) to download and install Ballerina.
+**SOQL & SOSL Operations**
 
-2. Create a Salesforce account and create a connected app by visiting [Salesforce](https://www.salesforce.com) 
-and obtain the following parameters:
-* Base URl (Endpoint)
-* Client Id
-* Client Secret
-* Access Token
-* Refresh Token
-* Refresh URL
+The `wso2/sfdc37` package contains operations that query using SOQL and search using SOSL. This allows for complex 
+operations using SObjects relationships.
 
-IMPORTANT: This access token and refresh token can be used to make API requests on your own account's behalf. 
-Do not share your access token, client secret with anyone.
+**Describe Operations**
 
-Visit [here](https://help.salesforce.com/articleView?id=remoteaccess_authenticate_overview.htm) 
-for more information on obtaining OAuth2 credentials.
+The `wso2/sfdc37` package contains operations that describe SObjects, organizational data, available resources, APIs, and 
+limitations for organizations.
 
-3. Create a new Ballerina project by executing the following command.
+## Compatibility
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta15 |
+|Salesforce REST API |    v37.0      |  
+
+## Sample
+The Salesforce connector can be used to create an Account SObject, execute a SOQL query, and create a Lead SObject. 
```

Comment:
Delete this sentence: "The Salesforce connector can be used to create an Account SObject, execute a SOQL query, and create a Lead SObject."

Reply:
Fixed